Dockerfile

```diff
@@ -278,6 +281,8 @@ RUN --mount=type=cache,sharing=locked,id=moby-dev-aptlib,target=/var/lib/apt \
             python3-pip \
             python3-setuptools \
             python3-wheel \
+            systemd \
+            systemd-sysv \
```

Comment:
Should we move the systemd-specific dependencies to the `dev-systemd-true` stage as well?

Reply:
moved